package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@guardian/braze-components",
-  "version": "5.0.0",
+  "version": "5.1.0-0",
```

Comment:
Should we also add the engines property to package.json with node 14 in it? { "engines" : { "node" : ">=0.14" } }

Reply:
Hey that's a good question. I'm not sure. 🤔  I guess we don't expect our consumers to use this package in the Node environment, but in the browser. Given that we don't specify it currently, I'm tempted to not make a change in this PR and investigate separately (maybe we have gu wide recommendations for this).